templates/probot/files/platformsh/setup/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+
+# Create a symlink for the .env file, where GitHub App credentials need to be written during registration hand shake.
+ln -s registration/.env .env
+
+# Remame user-supplied app.yml file, so that correct WEBHOOK_PROXY_URL can be written to it during deploy.
+mv app.yml probot.app.yml
+
+# Create a symlink for the app.yml file, which will need to be writable during deploy, but in project root.
+ln -s registration/app.yml app.yml
+
+# Install Platform.sh CLI, probably needs to have a check here so it isn't installed on every build.
```

Comment:
There's no build cache in use here, so it would need to be reinstalled every time.

```suggestion
# Install Platform.sh CLI.
```